services/udp/service.go

```diff
@@ -64,6 +66,7 @@ func NewService(c Config) *Service {
 	return &Service{
 		config:  d,
 		done:    make(chan struct{}),
+		bytebuf: make(chan []byte, 1000),
```

Comment:
+1 to `parserChan`

I really don't have any particular reason to set it to 1000. In my testing I rarely saw the length of this channel get beyond 10. So I thought I'd just go up two orders of magnitude, since 1000 isn't going to consume much memory at all and should allow it to handle very high loads with smaller packets. I will make it a constant.